pyo3-ffi/build.rs

```diff
@@ -112,9 +112,16 @@ fn ensure_python_version(interpreter_config: &InterpreterConfig) -> Result<()> {
                     .0
                     .contains(&BuildFlag::Py_GIL_DISABLED)
                 {
-                    warn!(
+                    let version = interpreter_config.version;
+                    if version < PythonVersion::PY313 {
+                        panic!(
```

Comment:
This should use `ensure!` rather than `panic!`, it leads to a friendlier presentation of the error to users.